src/test/java/com/amazon/ion/conformance/ConformanceTestRunner.kt

```diff
@@ -87,6 +87,12 @@ abstract class ConformanceTestRunner(
             "set_symbols does not accept null.string" in completeTestName -> false
             "set_symbols does not accept annotated arguments" in completeTestName -> false
 
+            // FIXME: Add syntax checks in MacroCompiler
+            "tdl/expression_groups.ion" in file.absolutePath -> false
+
+            // FIXME: Implicit rest args don't always work
+            "implicit rest args" in completeTestName -> false
```

Comment:
🗺️ Specifically, rest args for the `if_*` tests are failing. I haven't yet isolated whether this is because of an issue with those macros (unlikely), TDL in general (would also be surprising to me), or something else.